src/shared_utilities/Domains.jl

```diff
@@ -88,10 +88,47 @@ and not rely on any internal defaults set in ClimaCore.
 """
 function Point(;
     z_sfc::FT,
-    comms = ClimaComms.SingletonCommsContext(),
+    context = ClimaComms.SingletonCommsContext(),
 ) where {FT}
     coord = ClimaCore.Geometry.ZPoint(z_sfc)
-    space = (; surface = ClimaCore.Spaces.PointSpace(comms, coord))
+    space = (; surface = ClimaCore.Spaces.PointSpace(context, coord))
+    return Point{FT, typeof(space)}(z_sfc, space)
+end
+
+"""
+    Point(; z_sfc::FT, lat::FT, long::FT, context = ClimaComms.SingletonCommsContext())
+
+A constructor for the `Point` domain that creates a space with a latitude and longitude.
+This should be used for simulations at a point that require reading in spatial data
+from a file, such as the CLM data, which is defined on a lat/long grid.
+
+The latitude and longitude of the returned domain can be accessed as follows:
+- lat = domain.space.surface.local_geometry.coordinates.lat[]
+- long = domain.space.surface.local_geometry.coordinates.long[]
+"""
+# TODO when I use kwargs this constructor overwrites the other one - why?
+function Point(
+    z_sfc::FT,
+    lat::FT,
+    long::FT,
+    context = ClimaComms.SingletonCommsContext(),
+) where {FT <: AbstractFloat}
+    coord = ClimaCore.Geometry.LatLongZPoint(lat, long, z_sfc)
+    AIdx = ClimaCore.Geometry.coordinate_axis(coord)
+    local_geometry = ClimaCore.Geometry.LocalGeometry(
+        coord,
+        FT(1.0),
+        FT(1.0),
+        ClimaCore.Geometry.AxisTensor(
+            (
+                ClimaCore.Geometry.LocalAxis{AIdx}(),
+                ClimaCore.Geometry.CovariantAxis{AIdx}(),
+            ),
+            SMatrix{3, 3, FT}(1, 0, 0, 0, 1, 0, 0, 0, 1),
+        ),
+    )
```

Comment:
@dennisYatunin In ClimaLand we have a need for a PointSpace that has a lat/lon attached to it, so I'm constructing one here. It is correctly constructed with the lat/lon, but I wasn't sure of the best way to set up the AxisTensor so I made it like this. Does this look okay to you (given that it's just for a point space)?